src/modules/windows/feeds/FeedsContentsWidget.cpp

```diff
@@ -314,6 +314,12 @@ void FeedsContentsWidget::removeEntry()
 	}
 }
 
+void FeedsContentsWidget::markEntriesAsRead()
+{
+	m_feed->markAllEntriesAsRead();
```

Comment:
Downgrade to lambda used by `QMenu::addAction()`, add empty line before `updateFeedModel()` call and wrap it all in `if (m_feed)` check.